### PR TITLE
Migrate notifications view for Bootstrap UI

### PR DESCRIPTION
**Pull request by eduardoj**
Before:

screenshot from 2019-02-25 17-36-25_before

Now:

screenshot from 2019-02-25 17-36-53_now

### CODE REVIEW

**Comment by dmarcoux** [comment]
The flash message should be a `success` instead of a `notice`.

**Comment by dmarcoux** [comment]
The flash message should be a `success` instead of a `notice`.

https://github.com/openSUSE/open-build-service/blob/db9828363e0dda6fab5d53d11f172a5b87b68789/src/api/app/controllers/webui/subscriptions_controller.rb#L13

**Comment by eduardoj** [comment]
[Notifications](https://trello.com/c/dN9wAiMq/1054-notifications)

**Comment by bgeuken** [comment]
On the user notification page we have a Groups field at the top. It's currently not styled the same way like the rest of the notifications:



It was already like that, but would be cool if you could update this as well.

**Comment by eduardoj** [comment]
@dmarcoux:

> The flash message should be a `success` instead of a `notice`.

Done.

File: src/api/app/views/webui2/webui/subscriptions/index.html.haml [hunk 1]
```diff
@@ -4,3 +4,10 @@
 .card.mb-3
   = render partial: 'webui/configuration/tabs'
   .card-body
+    %h3.card-title
+      Events
+    %p.card-text
+      Choose default notification settings for events (checked means direct email).
+      = form_tag(notifications_path, method: :put, id: 'notification_form') do
+        = render partial: 'subscriptions_form'
+        %p= submit_tag 'Update'
```
[the remark this answers is not in the document]
Comment (eduardoj) [reply src/api/app/views/webui2/webui/subscriptions/index.html.haml]: @bgeuken:

> The button isn't styled

I will implement this form with ajax, in this same pull request. The button will be not needed. Marking as "DO NOT MERGE".

**Comment by eduardoj** [comment]
@bgeuken:

> On the user notification page we have a Groups field at the top. It's currently not styled the same way like the rest of the notifications:
>  [...]
> It was already like that, but would be cool if you could update this as well.

Done separately in #7046. Please have a look.

File: src/api/app/views/webui2/webui/subscriptions/index.html.haml (hunk 1 above)
[the remark this answers is not in the document]
Comment (eduardoj) [reply src/api/app/views/webui2/webui/subscriptions/index.html.haml]: > I will implement this form with ajax, in this same pull request. The button will be not needed. Marking as "DO NOT MERGE".

Well, in other pull request.

**Pull request merged**